src/app/content/studyGuides/reducer.ts

```diff
@@ -31,11 +34,34 @@ const reducer: Reducer<State, AnyAction> = (state = initialState, action) => {
       return {...state, summary: {...state.summary, open: false}};
     case getType(actions.loadMoreStudyGuides):
       return {...state, summary: {...state.summary, loading: true}};
+    case getType(actions.setSummaryFilters): {
+      const aux = {
+        ...state,
+        summary: {
+        ...state.summary,
+        filters: {
+          ...state.summary.filters,
+          ...action.payload,
+        },
+          loading: true,
+          pagination: null,
+          studyGuides: {},
+        },
+      };
+
+      console.log(aux);
+      return aux;
+    }
     case getType(actions.receiveStudyGuidesTotalCounts):
+      const locationIds = Array.from(action.meta.keys());
       return {
         ...state,
         summary: {
           ...state.summary,
+          filters: {
+            ...state.summary.filters,
+            locationIds,
```

Comment:
we don't have to set the filters when receiving the total counts, because we will set it to the current chapter when the dialog is opened, you can remove some of this logic